app/game/middleware/server-pipe.js

```diff
@@ -40,17 +40,17 @@ function getAction(message) {
         case MESSAGE_NAME.roundCountdownChanged:
             return RoundActionsCreator.updateRemaining(message.roundCountdown);
         default:
-            warn('Unknown message from server');
+            return null;
     }
-
-    return null;
 }
 
 function handleServerMessage(message, dispatch) {
```

Comment:
The "profit" here is that code would not be repeated. Right now it may not be the most urgent problem to solve, but it is certainly something that needs to be improved.